File: Python/Array/Largest Element in Array.py
//https://www.geeksforgeeks.org/problems/largest-element-in-array4009/1
from typing import List


class Solution:
    def largest(self, arr):
        # code here
        max_ = arr[0]
        for num in arr:
            if num > max_:
                max_ = num
        return max_

File: Cpp/Array/Largest Element in Array.cpp
//https://www.geeksforgeeks.org/problems/largest-element-in-array4009/1
class Solution {
  public:
    int largest(vector<int> &arr) {
        // code here
        int n=arr.size();
        int max=arr[0];
        for(int i=0;i<n;i++){
            if(arr[i]>max){
                max=arr[i];
            }
        }
        return max;
    }
};
